.cicd/docker/ubuntu-18.04.dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:18.04
+# install dependencies
+RUN apt-get update && \
+    apt-get install -y build-essential git automake python2.7 python2.7-dev python3 python3-dev curl ccache
+
```

Comment:
Let's make this Dockerfile look the same as the others, including comments and spacing.